src/netmanager-app/app/(authenticated)/sites/[id]/page.tsx

```diff
@@ -80,7 +79,7 @@ export default function SiteDetailsPage() {
               <DialogHeader>
                 <DialogTitle>Edit Site</DialogTitle>
                 <DialogDescription>
-                  Make changes to the site details here. Click save when you're
+                  Make changes to the site details here. Click save when you&apos;re
```

Comment:
Hi @Codebmk , is this ( `you&apos;re `) intentional?

Reply:
Yes adding an apostrophe directly is not acceptable in jsx ie returns type errors, because it is a special character